tensorflow/compiler/mlir/lite/transforms/optimize.cc

```diff
@@ -1161,7 +1198,7 @@ void OptimizePass::runOnFunction() {
       FuseBinaryOpToFollowingDepthwiseConv2D,
       FuseBinaryOpToFollowingFullyConnected, FuseConv2DAndMulWithQDQs,
       FuseDepthwiseConv2DAndMulWithQDQs, ConvertTrivialTransposeOpToReshapeOp,
-      RemoveReshapeBeforeFullyConnected>(ctx);
+      RemoveReshapeBeforeFullyConnected, ConvertPackToReshape>(ctx);
```

Comment:
Could you make this pattern with the canonicalization of tf.Pack operator instead of tfl.Pack operator?

Reply:
Sure, let me update it 😃